product-service-description/revisionssichere-daten-as-a-service/produkte/README.md

```diff
@@ -1,4 +1,44 @@
 # Revisionssichere Daten as a Service Produkte
 
+!(media/revisionsichere-daten-as-a-service.png)
+
+|  | POSArchiv | AKO |
+| --- | --- | --- |
+| Speichern der Belegdaten | X |  |
+| Speichern der Kassenabschlüsse | X | X |
+| revisionssichere Belege | X |  |
+| Belegsicherung in der Cloud | X | X |
+| tar-File Export | X |  |
+| DSFinV-K Export | X | X |
+| DATEV/DEFKA Export | X |  |
+
+
 - [POS Archiv](pos-archiv.md) 
-- [Audicon Kassenarchiv Online](Audicon-Kassenarchiv-Online.md) 
\ No newline at end of file
+
+  - *Revisionssicherer Speicher* für DSFinV-K Files (Tagesabschlüsse),
+TSE-TAR Files, PDF Files über 10 Jahre und
+Verfahrensdokumentation.
+  - *Hochladen der Daten* manuell oder über die zur Verfügung
+gestellte API.
+  - *Kundenbindung für Kassenhändler!* Kassenbetreiber speichert
+seine Daten über die Webseite des Kassenhändlers. Der
+Kassenhändler kann bei Bedarf auf diese Daten zugreifen und
+diese im Falle einer Steuerprüfung für den Kassenbetreiber exportieren.
+  - *Web-Portal* ermöglicht mandantenfähigen Zugriff und einfache Integration in die Webseite.
```

Comment:
Ist diese Änderung so korrekt? "- *Web-Portal* ermöglicht mandantenfähigen Zugriff und einfache Integration in eine beliebige Webseite."